### ISSUE TITLE
Can't see character under cursor

### ISSUE THREAD

**Issue by andyschweig** [XVimProject/XVim#674]
I just started using XVim. When in command mode, the cursor is a gray block, which makes it difficult to see the character under the cursor. Is there a way to change the appearance of the cursor? Thanks.
##

**Comment by Keithbsmiley** [comment]
Yep this should be changed in your colorscheme. This should be closed.

**Comment by aes42** [comment]
Thanks for your responses. I tried making the cursor a light gray and that does make the character underneath easier to see. The problem is that when the cursor is light like that, the thin line cursor in insert mode is almost impossible to see. I think the ideal behavior would be what happens in gvim or MacVim, which is that the block cursor is dark and the character underneath is shown in white. Is something like that possible? Thanks.
